### DISCUSSION TITLE
phpstan rule to throw error if ray() method call found

### DISCUSSION THREAD

**Discussion by pavarnos** [spatie/ray#414] [Ideas]
Random idea: is it possible to have a phpstan.neon file that will emit an error if `ray()` is seen anywhere in your code? This would make it much harder to accidentally commit code to production that called ray() if ray was in require-dev

**Comment by AdrianMrn** [comment]
@pavarnos check out this Twitter thread for some ideas people already came up with! https://twitter.com/freekmurze/status/1379766627412115457

**Reply by pavarnos** [reply]
Worked well. Thank you for the quick answer
